gen-app-builder/data-store-status-checker/examples/data_store_checker.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.            request = discoveryengine_v1.ListDocumentsRequest(
</pre><p>Can you change this to just edit <code>request.page_token</code> instead of recreating it? Will it still work? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
In theory that would work as well.  
I ended up doing a slight refactor that should just obfuscate this from he main method.  

The net-net is that a request does get rebuilt on each call, but it's a proto build so this should be negligible impact to the code.